docker-compose.yml

```diff
@@ -45,9 +45,37 @@ services:
     networks:
       - back-tier
 
+  kcpostgres:
+    image: postgres
+    volumes:
+      - kc-pg-data:/var/lib/postgresql/data
+    environment:
+      POSTGRES_DB: keycloak
+      POSTGRES_USER: keycloak
+      POSTGRES_PASSWORD: password
+  keycloak:
+      image: quay.io/keycloak/keycloak:latest
+      environment:
+        DB_VENDOR: POSTGRES
+        DB_ADDR: kcpostgres
+        DB_DATABASE: keycloak
+        DB_USER: keycloak
+        DB_SCHEMA: public
+        DB_PASSWORD: password
+        KEYCLOAK_USER: admin
+        KEYCLOAK_PASSWORD: Pa55w0rd
+        # Uncomment the line below if you want to specify JDBC parameters. The parameter below is just an example, and it shouldn't be used in production without knowledge. It is highly recommended that you read the PostgreSQL JDBC driver documentation in order to use it.
+        #JDBC_PARAMS: "ssl=true"
```

Comment:
Is JDBC something that's still used if Java isn't involved anywhere?

Reply:
I don't know I grabbed that change from Paul's PR. I assumed its for the keycloak implementation, as it uses env vars for configuration.